gitlab/resource_gitlab_service_slack.go

```diff
@@ -58,41 +57,68 @@ func resourceGitlabServiceSlack() *schema.Resource {
+				Type:        schema.TypeString,
 				Optional:    true,
-				Computed:    true,
 			},
-			"push_channel": {
-				Description: "The name of the channel to receive push events notifications.",
-				Type:        schema.TypeString,
+			"confidential_issues_events": {
+				Description: "Enable notifications for confidential issues events.",
+				Type:        schema.TypeBool,
 				Optional:    true,
+				Computed:    true,
 			},
-			"issues_events": {
-				Description: "Enable notifications for issues events.",
+			// TODO: Currently, GitLab ignores this option (not implemented yet?), so
+			// there is no way to set it. Uncomment when this is fixed.
+			// See: https://gitlab.com/gitlab-org/gitlab-ce/issues/49730
+			// "confidential_note_channel": {
+			// 	Description: "The name of the channel to receive confidential note events notifications.",
+			// 	Type:        schema.TypeString,
+			// 	Optional:    true,
+			// },
+			"confidential_note_events": {
+				Description: "Enable notifications for confidential note events.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Computed:    true,
 			},
+			// TODO: Currently, GitLab doesn't correctly implement the API, so this is
+			//       impossible to implement here at the moment.
+			//       see https://gitlab.com/gitlab-org/gitlab/-/issues/28903
+			// "deployment_channel": {
+			// 	Description: "The name of the channel to receive deployment events notifications.",
+			// 	Type:        schema.TypeString,
+			// 	Optional:    true,
+			// },
+			// "deployment_events": {
+			// 	Description: "Enable notifications for deployment events.",
+			// 	Type:        schema.TypeBool,
+			// 	Optional:    true,
+			// 	Computed:    true,
+			// },
 			"issue_channel": {
 				Description: "The name of the channel to receive issue events notifications.",
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
-			"confidential_issues_events": {
-				Description: "Enable notifications for confidential issues events.",
+			"issues_events": {
+				Description: "Enable notifications for issues events.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Computed:    true,
 			},
-			"confidential_issue_channel": {
-				Description: "The name of the channel to receive confidential issue events notifications.",
-				Type:        schema.TypeString,
-				Optional:    true,
-			},
-			"merge_requests_events": {
-				Description: "Enable notifications for merge requests events.",
+			// TODO: Currently, go-gitlab doesn't implement this option yet.
+			//       see https://github.com/xanzy/go-gitlab/issues/1354
+			"job_events": {
+				Description: "Enable notifications for job events. **ATTENTION**: This attribute is currently not being submitted to the GitLab API, due to https://github.com/xanzy/go-gitlab/issues/1354.",
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Computed:    true,
```

Comment:
Can you put `job_events` back to being just Computed, not Optional.

Reply:
done